File: natureremocontroller/src/main/java/com/fishwaffle/natureremo/controller/models/AirConRangeMode.java
/*
 * Copyright (c) 2018 FishWaffle.
 */

package com.fishwaffle.natureremo.controller.models;

import android.support.annotation.NonNull;

import java.io.Serializable;
import java.util.Arrays;

public class AirConRangeMode  implements Serializable {
    /**温度*/
    public String[] temp;
    /**風量*/
    public String[] vol;
    /**風向き*/
    public String[] dir;

    @NonNull
    @Override
    public String toString() {
        return "AirConRangeMode{" +
                "temp=" + Arrays.toString(temp) +
                ", vol=" + Arrays.toString(vol) +
                ", dir=" + Arrays.toString(dir) +
                '}';
    }
}

File: natureremocontroller/src/main/java/com/fishwaffle/natureremo/controller/models/AirConRangeMode.kt
/*
 * Copyright (c) 2019 FishWaffle.
 */

package com.fishwaffle.natureremo.controller.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fishwaffle.natureremo.controller.NatureRemo.isIgnoreUnknown
import java.io.Serializable

/**
 * @param temp 温度
 * @param vol 風量
 * @param dir 風向き
 */
@JsonIgnoreProperties(ignoreUnknown = isIgnoreUnknown)
data class AirConRangeMode(var temp: List<String>?, var vol: List<String>?, var dir: List<String>?) : Serializable